code/datums/wounds/burns.dm

```diff
@@ -0,0 +1,334 @@
+				condition += ", <span class='deadsay'>with streaks of rotten infection pulsating outward!</span>"
+			if(WOUND_INFECTION_SEPTIC to INFINITY)
+				return "<span class='deadsay'><B>[victim.p_their(TRUE)] [limb.name] is a mess of char and rot, skin literally dripping off the bone with infection!</B></span>"
+			else
+				condition += "!"
+
+	return "<B>[victim.p_their(TRUE)] [limb.name] [examine_desc][condition]</B>"
+
+/datum/wound/burn/get_scanner_description(mob/user)
+	. = ..()
+	. += "<div class='ml-3'>"
+
+	if(infestation <= sanitization && flesh_damage <= flesh_healing && mortification == 0)
+		. += "No further treatment required: Burns will heal shortly."
+	else
+		switch(infestation)
+			if(WOUND_INFECTION_MODERATE to WOUND_INFECTION_SEVERE)
+				. += "Infection Level: Moderate\n"
+			if(WOUND_INFECTION_SEVERE to WOUND_INFECTION_CRITICAL)
+				. += "Infection Level: Severe\n"
+			if(WOUND_INFECTION_CRITICAL to WOUND_INFECTION_SEPTIC)
+				. += "Infection Level: <span class='deadsay'>CRITICAL</span>\n"
+			if(WOUND_INFECTION_SEPTIC to INFINITY)
+				. += "Infection Level: <span class='deadsay'>LOSS IMMINENT</span>\n"
+
+		if(flesh_damage > 0)
+			. += "Flesh damage detected: Please apply ointment or regenerative mesh to allow recovery.\n"
+
+		if(mortification > 0)
+			. += "Mortification detected: Please excise dead skin via debriding surgery. If desperate, dead skin can be shaved off by applying an aggressive hold and interacting helpfully with a sharp object on the affected limb."
+	. += "</div>"
+
+/*
+	new burn common procs
+*/
+
+// TODO:actually balance ointment + regen mesh + bandages
+/datum/wound/burn/proc/ointment(obj/item/stack/medical/ointment/I, mob/user)
+	user.visible_message("<span class='notice'>[user] begins applying [I] to [victim]'s [limb.name]...</span>", "<span class='notice'>You begin applying [I] to [user == victim ? "your" : "[victim]'s"] [limb.name]...</span>")
+	if(!do_after(user, (user == victim ? I.self_delay : I.other_delay), extra_checks = CALLBACK(src, .proc/still_exists)))
+		return
+
+	limb.heal_damage(I.heal_brute, I.heal_burn)
+	user.visible_message("<span class='green'>[user] applies [I] to [victim].</span>", "<span class='green'>You apply [I] to [user == victim ? "your" : "[victim]'s"] [limb.name].</span>")
+	I.use(1)
+	sanitization += I.sanitization
+	flesh_healing += I.flesh_regeneration
+
+	if((infestation <= 0 || sanitization >= infestation) && (flesh_damage <= 0 || flesh_healing > flesh_damage))
+		to_chat(user, "<span class='notice'>You've done all you can with [I], now you must wait for the flesh on [victim]'s [limb.name] to recover.</span>")
+	else
+		try_treating(I, user)
+
+/datum/wound/burn/proc/bandage(obj/item/stack/medical/gauze/I, mob/user)
+	if(current_bandage)
+		if(current_bandage.absorption_capacity > I.absorption_capacity + 1)
+			to_chat(user, "<span class='warning'>The [current_bandage] on [victim]'s [limb.name] is still in better condition than your [I.name]!</span>")
```

Comment:
the check is against I.absorption_capacity + 1, so isn't this message a bit misleading?